src/lib/utils.ts

```diff
@@ -19,6 +19,20 @@ export function toUpperCase(value: string): string {
   return (value || "").toUpperCase();
 }
 
+/**
+ * Convenience function for making an enum value, that could be nullish, user friendly i.e. underscores removed and title-cased.
+ *
+ * @param value {string}   The string value; could be null
+ * @returns     {string}   The string value made user friendly
+ */
+export function makeEnumValueUserFriendly(value: string): string {
+  if (value) {
+    value = value.replace(/_/g, " ");
+    return value[0] + value.slice(1).toLowerCase();
+  }
+  return value;
+}
```

Comment:
Like it, definitely prefer the functional approach over having a map in this case!  

What are the cases where `value === null`?

Also, tiny nitpick but I believe what's happening at line 31 changes value to be sentence-case, rather than title-case (TSdoc at line 23).

Reply:
🤯 @ sentence case, hadn't heard of that before but yeah that makes more sense, thank you!

and when value is null, it just returns null - which then is displayed as `no data` on the Check Answers page. 😬